terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,56 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
+    if config.deployment_stage == 'anvildev':
+        broad_deployment = 'dev'
+    elif config.deployment_stage == 'anvilprod':
+        broad_deployment = 'prod'
+    else:
+        assert False, config.deployment_stage
+    return ' '.join([
+        "default-src 'self';",
+        (
+            "img-src 'self' data:"
+            " https://lh3.googleusercontent.com"
+            " https://www.google-analytics.com"
+            " https://www.googletagmanager.com;"
+        ),
+        (
+            "script-src 'self' 'unsafe-inline' 'unsafe-eval'"
+            " https://accounts.google.com/gsi/client"
+            " https://www.google-analytics.com"
+            " https://www.googletagmanager.com;"
+        ),
+        (
+            "style-src 'self' 'unsafe-inline'"
+            " https://fonts.googleapis.com"
+            " https://use.typekit.net"
+            " https://p.typekit.net;"
+        ),
+        (
+            "font-src 'self' data:"
+            " https://fonts.gstatic.com"
+            " https://use.typekit.net/af/;"
+        ),
+        "object-src 'none';",
+        (
+            "connect-src 'self'"
+            " https://www.googleapis.com/oauth2/v3/userinfo"
+            " https://www.google-analytics.com"
+            " https://www.googletagmanager.com"
+            " https://support.terra.bio/api/v2/"
+            f" https://sam.dsde-{broad_deployment}.broadinstitute.org/register/user/v1"
+            f" https://firecloud-orchestration.dsde-{broad_deployment}.broadinstitute.org/api/nih/status"
```

Comment:
Let's add one for the orchestration endpoint as well. We will likely have to interact with it soon anyways.